### PR TITLE
Apex content

### LINKED ISSUE #55: Configs - Apex
We have *some* content already added to the configs: https://github.com/AsYetUntitled/Framework/tree/master/ApexContent. However it's lacking, and we need more **appropriate** things added.

Anyone up for it ;)?

P.S. Don't forget BattlEye

### PR DESCRIPTION
**Pull request by jcbjoe**
Added pretty much all the needed apex content. The overpowered and
unessasary things arnt in I.E Lynx.
- Vehicles
- Clothing and Hats
- Weapons - Damn PM 9mm mag

This has got the humming bird commits in also as i have seemed to messup my commits. I have it sorted now and wont happen next time. I just cant get it removed from this branch

Thanks to WIlDott for helping me

Closes #55

### CODE REVIEW

**Comment by tkcjesse** [comment]
Can you revert your hummingbird stuff please. Please and thankyou. Or just do a new commit that reverts it. We'll squash.

**Comment by jcbjoe** [comment]
Reverted

File: ApexContent/Config_Vehicles.hpp [hunk 1]
```diff
@@ -167,39 +168,47 @@ class LifeCfgVehicles {
         price = -1;
         textures[] = {};
     };
-    
+
+    // Apex DLC
+    class C_Boat_Transport_02_F { //NEEDS TO BE CHANGED
+        vItemSpace = 200;
+        licenses[] = { {""}, {""}, {""}, {""} };
+        price = -1;
```
Comment (BoGuu) [inline ApexContent/Config_Vehicles.hpp]: These etc

**Comment by BoGuu** [comment]
Could you define prices for the vehicle configs? Then will happily merge

**Comment by jcbjoe** [comment]
Added prices

**Pull request merged**